http/jaxrs/src/main/java/io/quarkus/ts/security/core/GreetingResource.java

```diff
@@ -1,17 +1,127 @@
 package io.quarkus.ts.security.core;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 
-@Deprecated
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
 @Path("/hello")
 public class GreetingResource {
 
+    private final ObjectMapper MAPPER = new JsonMapper();
+
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String hello() {
         return "hello";
     }
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/serialize-string")
+    public String serstring() throws IOException, ClassNotFoundException {
+
+        byte[] bytes;
+
+        try (
+                ByteArrayOutputStream baos = new ByteArrayOutputStream();
+                ObjectOutputStream oos = new ObjectOutputStream(baos)) {
+
+            oos.writeObject("Hello RESTEasy");
+            oos.flush();
+            bytes = baos.toByteArray();
+        }
+
+        try (ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
+                ObjectInputStream ois = new ObjectInputStream(bais)) {
+            return (String) ois.readObject();
+        }
+    }
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/serialize-list")
+    public String serlist() throws IOException, ClassNotFoundException {
+
+        byte[] bytes;
+
+        try (
+                ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Can we use jdkSerialize and jdkDeserialize in all methods of this class?

Reply:
We have an issue with the native mode that is under investigation. Have a look at the description of this ticket. I think that maybe in the future `jdkSerialize` and `jdkDeserialize` are going to be removed.